File: exercise09/paint_calculator.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)

	fmt.Print("Enter the length in feet? ")
	scanner.Scan()
	length, _ := strconv.Atoi(scanner.Text())

	fmt.Print("Enter the width in feet? ")
	scanner.Scan()
	width, _ := strconv.Atoi(scanner.Text())

	fmt.Printf("You will need to purchase %v gallons of\npaint to cover %v square feet.\n", (length*width)/350, length*width)

}

File: javascript/paint-calc/paint-calculator.js
import { input } from "pynput";

const length = Number(await input("Enter the length in feet? "));
const width = Number(await input("Enter the width in feet? "));

console.log(
	`You will need to purchase ${(length * width) / 350} gallons of\npaint to cover ${length * width} square feet.`,
);
